_docs/_latest/configure/notifier/index.md

```diff
@@ -29,6 +29,63 @@ and varying formats to you to events in your Google Cloud Platform
 
 ## Configuring Notifier
 
+### Email Connector Config
+
+Forseti security provides an interface to add the email connector of your
+choice. A new section named `email_connector` has been introduced under 
+`notifier` as part of the new design. The connector details provided in this
+section will be used to send out inventory summary and violations via email
+when found.notify users violations when found for the enabled
+scanners. 
+
+To configure `email_connector`, follow the steps below:
+
+1. Open `forseti-security/configs/server/forseti_conf_server.yaml`.
+2. Navigate to the `notifier` > `email_connector` section.
+
+If you want the notifier to send violations and/or inventory summary via email, 
+provide valid values for all the fields mentioned below.
```

Comment:
"provide the corresponding values" might read better?

Reply:
Updated